File: common/src/test/java/com/hartwig/actin/util/AminoAcidTest.java
package com.hartwig.actin.util;

import static org.junit.Assert.assertEquals;

import org.junit.Test;

public class AminoAcidTest {

    @Test
    public void canConvertAminoAcidsToSingleLetter() {
        assertEquals("BRAF p.V600E", AminoAcid.forceSingleLetterAminoAcids("BRAF p.Val600Glu"));
        assertEquals("BRAF p.V600E", AminoAcid.forceSingleLetterAminoAcids("BRAF p.V600E"));
    }
}

File: molecular/src/test/kotlin/com/hartwig/actin/molecular/orange/AminoAcidTest.kt
package com.hartwig.actin.molecular.orange

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class AminoAcidTest {

    @Test
    fun `Should convert amino acids to single letters`() {
        assertThat(AminoAcid.forceSingleLetterAminoAcids("p.Val600Glu")).isEqualTo("p.V600E")
        assertThat(AminoAcid.forceSingleLetterAminoAcids("p.V600E")).isEqualTo("p.V600E")
    }
}